files-backup.php

```diff
@@ -29,11 +29,12 @@
 
 // Make backup.
 exec(
-  "tar czf files-$date.tar.gz -C $backdrop_root files/ &&
-  mkdir -p $destination/files_backups
-  mv files-$date.tar.gz $destination/files_backups"
+  "tar czf $db_name-files-$date.tar.gz -C $backdrop_root files/ &&
+  mkdir -p $destination/files_backups;"
 );
 
+//mv files-$date.tar.gz $destination/files_backups"
```

Comment:
Is this intentional?  seems we are not moving the backup to the intended location w/out this.

Reply:
It was giving an error if I recall correctly but that was a long time ago. It is working now with the code omitted  but the cron job is partially taking care of the  file path.